config/01-ce-router-defaults.conf

```diff
@@ -106,6 +106,14 @@ SCHEDD_ATTRS = $(SCHEDD_ATTRS) HTCondorCEVersion grid_resource
 #   JOB_ROUTER_DEFAULTS = $(JOB_ROUTER_DEFAULTS) [set_foo = 1;]
 #
 MERGE_JOB_ROUTER_DEFAULT_ADS=True
+#!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+# Warning: JOB_ROUTER_DEFAULTS, JOB_ROUTER_ENTRIES, JOB_ROUTER_ENTRIES_CMD, and
+#          JOB_ROUTER_ENTRIES_FILE are deprecated and will be removed for V24 of
+#          HTCondor. New configuration syntax for the job router is defined using
+#          JOB_ROUTER_ROUTE_NAMES and JOB_ROUTER_ROUTE_<name>. For new syntax example visit:
+# https://htcondor.readthedocs.io/en/latest/grid-computing/job-router.html#an-example-configuration
+#   Note: The removal will occur during the lifetime of the HTCondor V23 feature series.
+#!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
```

Comment:
"will be removed for V24 of HTCondor" should be reworded to one of the following:

- "will be removed for V24 of HTCondor and HTCondor-CE"
- "will be removed for V24 of the HTCondor Software Suite"